flashinfer/trace/solutions/gelu_tanh_and_mul/native.py

```
@@ -0,0 +1,39 @@
+from flashinfer.activation import gelu_tanh_and_mul as _api
+from flashinfer.trace.solutions._helpers import solution_autotune
+
+definition = "gelu_tanh_and_mul"
+api = "flashinfer.activation.gelu_tanh_and_mul"
+backend = "flashinfer"
+inputs = ("input",)
+outputs = ("output",)
+api_kwargs = {"input": "input"}
+
+
+def run(input):
```

Ruff (0.16.1)

[error] Function argument `input` is shadowing a Python builtin

(A002)